python/paddle/nn/functional/loss.py

```diff
@@ -2229,3 +2229,105 @@ def hinge_embedding_loss(input, label, margin=1.0, reduction='mean', name=None):
         return paddle.sum(loss, name=name)
     elif reduction == 'none':
         return loss
+
+
+def cosine_embedding_loss(input1, input2, label, margin=0, reduction='mean'):
```

Comment:
the parameters of `x1, x2, target` in `paddle.nn.functional.cosine_embedding_loss`  [RFC](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20220319_api_design_for_CosineEmbeddingLoss.md) must be same with parameters here.

Reply:
According to the design requirements, our API requires the `name` parameter like other loss API.

Reply:
Now changing parameters will lead to the modification of code and document. Can I just change `x1, x2, target` to `input1, input2, label` in [RFC](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20220319_api_design_for_CosineEmbeddingLoss.md)?